styles/scss/_h5peditor-drag-and-drop.scss

```diff
@@ -546,4 +555,10 @@
   &:before {
     content: '\e85c';
   }
+}
+
+@media (max-width: 570px) {
```

Comment:
Get the breakpoint adapted please (smaller? I never know 😂)

https://github.com/h5p/h5p-editor-php-library/assets/55385809/503e592a-b91f-4a06-8263-717671a30a3b

Reply:
Fixed